tests/conftest.py

```diff
@@ -45,15 +46,39 @@ def pytest_configure():
         cprint(80 * "=", "red", attrs=["blink"])
 
 
+@contextlib.contextmanager
+def set_export_data_inside_rms_flag():
+    old = ExportData._inside_rms
+    ExportData._inside_rms = True
+    try:
+        yield
+    finally:
+        ExportData._inside_rms = old
+
+
+@contextlib.contextmanager
+def set_environ_inside_rms_flag():
+    unset = object()
+    old = os.environ.get("INSIDE_RMS", unset)
```

Comment:
Interesting! For uniqueness?

Reply:
Yeah, just in case its set to `None`. If i do not use a sentinel value i want be able do differentiate between the two.